cli/_tools/generate_data.ts

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env -S deno run --allow-net --allow-read --allow-write
+// @ts-nocheck Trust me
```

Comment:
I think this should be a TODO comment `TODO: fix type errors`

Reply:
Done